components/scripts/maven/network.settings

```diff
@@ -1,16 +1,19 @@
-# Uncomment and update the lines below to configure the build validation scripts to use a HTTP proxy when fetching build scan data.
-
+# Uncomment and update use a HTTP proxy when fetching build scan data.
 #http.proxyHost=proxy.host.com
 #http.proxyPort=8888
 #http.nonProxyHosts=localhost|127.*|[::1]
 #http.proxyUser=user
 #http.proxyPassword=password
-
 #https.proxyHost=proxy.host.com
 #https.proxyPort=8443
 #https.nonProxyHosts=localhost|127.*|[::1]
 #https.proxyUser=user
 #https.proxyPassword=password
 
-# Uncomment to disable SSL certificate validation
+# Uncomment to disable SSL certificate validation for the Gradle Enterprise server
 #allowUntrustedServer=true
+
+# Uncomment and update to use a custom trust store
+#javax.net.ssl.trustStore=myTrustStore.jks
+#javax.net.ssl.trustStorePassword=changeit
+#javax.net.ssl.trustStoreType=jks
```

Comment:
❓ Does OkHttp use these without any further configuration?

Reply:
Yes. It took a bit of effort to test this, but I verified it works.

The `HandshakeCertificates.Builder.addPlatformTrustedCertificates` method fetches the JVM's default trust store and adds all of the certificates it finds in it to the TrustManager that it creates for use by OkHTTP.  The JVM's default trust store is influenced by these system properties.